modules/ROOT/pages/clustering/settings.adoc

```diff
@@ -45,22 +45,22 @@ Possible values are:
 
 [.compact]
 `LIST`::
-Treats `dbms.cluster.discovery.endpoints` as a list of addresses of servers to contact for discovery.
+Treats `dbms.cluster.discovery.endpoints` (as of Neo4j 5.23, `dbms.cluster.discovery.v2.endpoints`) as a list of addresses of servers to contact for discovery.
```

Comment:
With all of these, it's not quite true to say 'as of 5.23', it's more 'if using V2 discovery, available as of 5.23,'